actors/builtin/miner/miner_state_test.go

```diff
@@ -761,20 +761,30 @@ func TestSectorNumberAllocation(t *testing.T) {
 	})
 }
 
-func TestPenalizationFundsInPriorityOrder(t *testing.T) {
+func TestRepayDebtInPriorityOrder(t *testing.T) {
 	harness := constructStateHarness(t, abi.ChainEpoch(0))
 
 	currentBalance := abi.NewTokenAmount(300)
 	fee := abi.NewTokenAmount(1000)
-	_, _, err := harness.s.PenalizeFundsInPriorityOrder(harness.store, abi.ChainEpoch(0), fee, currentBalance)
+	err := harness.s.ApplyPenalty(fee)
 	require.NoError(t, err)
 
+	assert.Equal(t, harness.s.FeeDebt, fee)
+	penaltyFromVesting, penaltyFromBalance, err := harness.s.RepayPartialDebtInPriorityOrder(harness.store, abi.ChainEpoch(0), currentBalance)
```

Comment:
Nit: need more tests here later for non-zero locked funds.

Reply:
Tracked in #966